pages/common/mamba.md

```diff
@@ -0,0 +1,37 @@
+# mamba
+
+> Fast, cross-platform package manager, intended as a drop-in replacement for conda.
+> Some subcommands such as `mamba repoquery` have their own usage documentation.
+> More information: <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html>.
+
+- Create a new environment, installing the specified packages into it:
+
+`mamba create --name {{environment_name}} {{python=3.10 matplotlib}}`
+
+- Update all packages in environment:
+
+`mamba update --all`
+
+- Search for a particular package across repositories:
+
+`mamba repoquery search {{numpy}}`
+
+- List all environments:
+
+`mamba info --envs`
+
+- Remove unused [p]ackages and [t]arballs from cache:
+
+`mamba clean -pt`
+
+- Activate an environment:
+
+`mamba activate {{environment_name}}`
+
+- List all installed packages in the currently activated environment:
+
+`mamba list`
+
+- Install packages into the current environment, specifying the package [c]hannel:
```

Comment:
Please move install up to second position after create, it's very often used

Reply:
Done!